mutatest/cli.py

```diff
@@ -222,6 +227,12 @@ def cli_parser() -> argparse.ArgumentParser:
     parser.add_argument(
         "--nocov", action="store_true", help="Ignore coverage files for optimization."
     )
+    parser.add_argument(
+        "--timeout_factor",
+        help="If the tests take this much longer than expected, they are aborted.",
+        default=5,
+        type=int,
```

Comment:
The help string may be more clear with "If a mutation trial running time is beyond this factor multiplied by the first clean trial running time then that mutation trial is aborted and logged as a timeout." 

What is the rationale for the `int` type? I would have assumed a value like `1.2` would be valid if I wanted to tolerate only a 20% time increase. Other numeric arguments use actions like `PositiveIntegerAction`, so we'll want a valid type action defined here - likely floats greater than 1.